python/sparkdl/transformers/tf_image.py

```diff
@@ -99,18 +102,12 @@ def getGraph(self):
         return self.getOrDefault(self.graph)
 
     def getInputTensor(self):
-        tensor_or_name = self.getOrDefault(self.inputTensor)
-        if isinstance(tensor_or_name, tf.Tensor):
-            return tensor_or_name
-        else:
-            return self.getGraph().get_tensor_by_name(tensor_or_name)
+        tensor_name = self.getOrDefault(self.inputTensor)
```

Comment:
I'm confused why this logic was changed. 
1/ does tf.Graph.get_tensor_by_name(tf.Tensor) return the Tensor?
2/ let's not change files that aren't being touched at all by the main objective of the PR as it increases the review time unnecessarily. you can make mini PRs for these if you really want.

Reply:
1. `tf.Graph.get_tensor_by_name(tensor_name)` returns the tensor.  But it requires the tensor name, not the operation name. It eventually follows [this code path](https://github.com/tensorflow/tensorflow/blob/r1.3/tensorflow/python/framework/ops.py#L2771-L2782).
2. This kind of "converter-return-single-type" motif has been requested several times in the past PR comments. The changes needed to enforce it is rather small and not touching any core logic.

Reply:
I see what bug it was fixing, but in the future PLEASE put such unrelated changes in another PR. This kind of distraction causes delays in getting the PR in, as well as cause additional difficulties in debugging later on when trying to identify which PR a bug may come from. For example, most of my comments in this review round are about this file. In the future, I will ask you to break up the PR in such cases.